src/sonic-yang-models/yang-models/sonic-mclag.yang

```diff
@@ -0,0 +1,171 @@
+module sonic-mclag {
+	namespace "http://github.com/Azure/sonic-mclag";
+	prefix smclag;
+	yang-version 1.1;
+
+
+    import ietf-yang-types {
+        prefix yang;
+    }
+
+	import ietf-inet-types {
+		prefix inet;
+	}
+
+    import sonic-types {
+        prefix stypes;
+    }
+
+    import sonic-port {
+        prefix prt;
+    }
+
+    import sonic-portchannel {
+        prefix spc;
+    }
+
+    import sonic-vlan {
+        prefix svlan;
+    }
+
+	organization
+		"SONiC";
+
+	contact
+		"SONiC";
+
+	description
+		"SONIC MCLAG";
+
+	revision 2021-07-28 {
```

Comment:
One PR should have only one revision field. We do not need separate field for ""Addressed review comments". Thx

Reply:
done